packages/form/src/form/components/ListboxComponent.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import AddOutlinedIcon from '@mui/icons-material/AddOutlined'
+import { VariableSizeList, ListChildComponentProps } from 'react-window'
+import { JsxElement } from 'typescript'
+import isEmpty from 'lodash/isEmpty'
+import get from 'lodash/get'
+import { Stack, Typography } from '@gravis-os/ui'
+import { getIsCreateOption } from '../utils/getIsCreateOption'
+import { useResetCache } from '../VirtualizedAutocompleteList.tsx/hooks/useResetCache'
+
+const LISTBOX_PADDING = 8
```

Comment:
This is a constant.

Also we shouldn't have this component here. It should be in the VirtualizedAutocomplete folder